### DISCUSSION TITLE
When the plugin is available for 5.2~

### DISCUSSION THREAD

**Discussion by Ender0920** [Sixze/ALS-Refactored#303] [General]
When the plugin is available for 5.2~

**Comment by Ender0920** [comment]
can`t build myself
"Since 5.2 parameters are no longer allowed for RIGVM_METHOD functions."

**Comment by ameaninglessname** [comment]
Just using the '5.2' branch

**Comment by Ender0920** [comment]
Yes,found it,thanks very much~~




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Sixze/ALS-Refactored"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月13日(星期六) 上午9:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Sixze/ALS-Refactored] When the plugin is available for 5.2~ (Discussion #303)





 
Just using the '5.2' branch

**Reply by ameaninglessname** [reply]
You're welcome 😄. Could you mark this question solved?

**Comment by Ender0920** [comment]
Hi,
Sorry to bother you again,it seems that all the sound wav file are invailided in UE5.2 ,any idea how to fix that? please give me a hint~~




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Sixze/ALS-Refactored"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月13日(星期六) 上午9:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Sixze/ALS-Refactored] When the plugin is available for 5.2~ (Discussion #303)





 
Just using the '5.2' branch

**Comment by Ender0920** [comment]
sure~




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Sixze/ALS-Refactored"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月13日(星期六) 下午4:57
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Sixze/ALS-Refactored] When the plugin is available for 5.2~ (Discussion #303)





 
You're welcome 😄. Could you mark this question solved?
